mxd/README.md

```diff
@@ -974,7 +974,216 @@ That's because, Trudy had created the [access policy](#762-create-access-policy)
 Can you create another access policy and contract definition which allows Bob to view Trudy's catalog?  
 `Hint:` Create an access policy using Bob's BPN Number (`BPNL000000000002`) and use this access policy id while creating contract definition.
 
-## 8. Improving the setup
+## 8. S3 to S3 Transfer
+
+After the terraform command has successfully completed minio console will look like below.
+
+![img.png](assets/dashboard.png)
+
+![img.png](assets/alice_bucket.png)
+
+Create asset metadata for provider connector i.e Alice.
+
+#### 8.1 Create Asset
+Create an asset using the following `curl` command:
+```shell
+curl --location 'http://localhost/alice/management/v2/assets' \
+--header 'Content-Type: application/json' \
+--header 'X-Api-Key: password' \
+--data-raw '{
+    "@context": {},
+    "asset": {
+        "@type": "Asset",
+        "@id": "3",
+        "properties": {
+            "description": "Product EDC Demo Asset 3- S3"
+        }
+    },
+    "properties": {
+        "version": "1.0",
+        "type": "AmazonS3",
+        "name": "document_alice",
+        "id": "document_alice",
+        "contenttype": "text/plain"
+    },
+    "dataAddress": {
+        "@type": "DataAddress",
+        "secretAccessKey": "qwerty123",
+        "accessKeyId": "qwerty123",
+        "endpointOverride": "http://10.96.103.86:9000",
```

Comment:
Is `endpointOverride` needed. It is already defined as environment variable.

Reply:
Removed endpointOverride.